### DISCUSSION TITLE
Implement a 'not-sure' option

### DISCUSSION THREAD

**Discussion by SagevdBrand** [asreview/asreview#568] [Ideas]
Would it be possible to have some kind of 'I don't know' button instead of only 'relevant' and 'irrelevant'? Sometimes you are not sure yet whether or not the abstract is relevant, but I don't want to add noise to my model by simply marking it as relevant.

**Comment by J535D165** [comment]
Yes, this is a great idea. It is on our todo list #328

**Comment by MartijnUX** [comment]
I would suggest the term: 'Skip' instead of 'not-sure'. @J535D165 What happens in de Machine Learning model if you press a 'not sure' button? 

This is not the first time that users come with this issue/question https://github.com/asreview/asreview/issues/387

What does it mean if we introduce this option? Elas should help the researcher make the right decision. If we add a ‘skip’ function, we don’t know why the user does this.
